python_modules/dagster-graphql/dagster_graphql/schema/schedules/schedules.py

```diff
@@ -96,19 +104,20 @@ def resolve_partition_set(self, graphene_info: ResolveInfo):
             external_partition_set=external_partition_set,
         )
 
-    def resolve_futureTicks(self, _graphene_info, **kwargs):
-        cursor = kwargs.get(
-            "cursor", get_timestamp_from_utc_datetime(get_current_datetime_in_utc())
+    def resolve_futureTicks(self, _graphene_info: ResolveInfo, **kwargs: Dict[str, Any]):
```

Comment:
I recommend `object` for `kwargs` values, it will enforce casting to the correct type